gigadb/app/tools/files-metadata-console/gitlab-config-ci-test.yml

```
@@ -42,4 +42,6 @@ FilesMetaDataConsoleTestRun:
   artifacts:
     untracked: true
     when: always
-    expire_in: 1 week
\ No newline at end of file
+    expire_in: 1 week
+  tags:
+    - $GITLAB_USER_LOGIN
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)